src/header/header.scss

```diff
@@ -13,7 +13,8 @@
   user-select: none;
   box-sizing: border-box;
   // iPhone notch support
-  padding: env(safe-area-inset-top) env(safe-area-inset-right) 0 env(safe-area-inset-left);
+  padding: env(safe-area-inset-top) env(safe-area-inset-right) 0
```

Comment:
I wonder why Greta PRs always fix formatting? @FlyersPh9

Reply:
These formatting fixes are from Prettier and need to be done manually because we disabled autoformat (it was messing with Stylelint).

So Jon should always format before pushing.